ansible_rulebook/builtin.py

```diff
@@ -555,6 +558,11 @@ def cancel_callback():
                     verbosity=verbosity,
                     event_handler=event_callback,
                     cancel_callback=cancel_callback,
+                    inventory=os.path.join(
+                        private_data_dir,
+                        "inventory",
+                        os.path.basename(inventory),
```

Comment:
Is there a reason to use a nested folder instead of directly "inventory" dir?

Reply:
@Alex-Izquierdo I am following the directory structure that they recommend for ansible-runner input
https://ansible.readthedocs.io/projects/runner/en/stable/intro/#runner-input-directory-hierarchy

Reply:
Ok,